x-pack/legacy/plugins/ml/public/license/check_license.tsx

```diff
@@ -4,51 +4,45 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-
 import React from 'react';
+// @ts-ignore No declaration file for module
+import { banners } from 'ui/notify';
+// import chrome from 'ui/chrome';
+import { EuiCallOut } from '@elastic/eui';
+// @ts-ignore No declaration file for module
 import { xpackInfo } from '../../../xpack_main/public/services/xpack_info';
-import { banners, addAppRedirectMessageToUrl } from 'ui/notify';
 import { LICENSE_TYPE } from '../../common/constants/license';
 import { LICENSE_STATUS_VALID } from '../../../../common/constants/license_status';
 
-import chrome from 'ui/chrome';
-import { EuiCallOut } from '@elastic/eui';
-
 let licenseHasExpired = true;
-let licenseType = null;
-let expiredLicenseBannerId;
+let licenseType: LICENSE_TYPE | null = null;
+let expiredLicenseBannerId: any;
```

Comment:
is it possible to provide a type?

Reply:
Adding type of `string` but it's worth noting that it's a guess because `banners` is js.
It's also worth noting that the value `expiredLicenseBannerId` is never really used, it's only checked to see if it has been set.